llm/llm.go

```diff
@@ -66,10 +66,13 @@ func New(model string, adapters []string, opts api.Options) (LLM, error) {
 		}
 	}
 
-	switch ggml.ModelFamily() {
-	case ModelFamilyLlama:
+	switch mf.Name() {
+	case "gguf":
+		opts.NumGQA = 0 // TODO: remove this when llama.cpp runners differ enough to need separate newLlama functions
```

Comment:
This is the one difference in the interface between the current mainline llama.cpp and the ggml runner. Im keeping them on the same running logic in hopes that the interface will stay roughly the same. We will see if that changes.